ccm_web/client/src/components/Alert.tsx

```diff
@@ -49,11 +49,11 @@ export default function Alert (props: AlertProps): JSX.Element {
   const { title, icon, embedded, children } = props
 
   const core = (
-    <Paper className={`${classes.dialog} ${classes.padding}`} role='alert'>
```

Comment:
Why is `StyledCorePare` component used?

Reply:
I see you have wrapped this with Paper. Can you tell me what was the need for this?

Reply:
This component sometimes returns core (just the paper component) by itself (in the case of an embedded alert), and in those situations the styling wasn't working so I manually styled the Paper component with emotion. From what I saw it properly styled the returned alert in these cases. Just a bug with the codemod that didn't handle this case